### ISSUE TITLE
Replace any outdated APIM versions that are mentioned within the doc space

### ISSUE THREAD

**Issue by ashera96** [wso2/docs-apim#8516]
**Description:**

Find and replace outdated APIM versions that have been specified within the docs. Sample scenarios that need to be identified while doing so are:

- https://apim.docs.wso2.com/en/latest/install-and-setup/setup/api-controller/enabling-correlation-logs-with-apictl/ (we can completely remove any mentions such as these)
- https://apim.docs.wso2.com/en/4.4.0/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#add-an-environment
- https://apim.docs.wso2.com/en/4.4.0/install-and-setup/setup/sso/okta-as-an-external-idp-using-saml/#prerequisites

Basically, migration-related scenarios/docs can specify the APIM version. Any other mentions of APIM versions should be removed from our docs to ensure maintainability down the line.
